acceptance/lib/pxp-agent/test_helper.rb

```diff
@@ -415,9 +415,9 @@ def wait_for_sleep_process(target)
   begin
     ps_cmd = target['platform'] =~ /win/ ? 'ps -efW' : 'ps -ef'
     sleep_process = target['platform'] =~ /win/ ? 'PING' : '/bin/sleep'
-    retry_on(target, "#{ps_cmd} | grep '#{sleep_process}' | grep -v 'grep'", {:max_retries => 15, :retry_interval => 2})
+    retry_on(target, "#{ps_cmd} | grep '#{sleep_process}' | grep -v 'grep'", {:max_retries => 120, :retry_interval => 1})
```

Comment:
We could make this even longer. With this commit, we allow ~120 seconds. If a host was running incredibly slow and took 121 seconds, we would still want the test to pass.
At the same time, I think we've had other platforms fail where the sleep process is incorrectly killed; and a long timeout would cause those platforms to take a long time to fail.

I think we could increase this to 5 or 10 minutes?